<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebuglogWindow</class>
 <widget class="QDialog" name="DebuglogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug Log</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_log">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>451</height>
    </rect>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>429</y>
     <width>206</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_level">
      <property name="text">
       <string>Min. severity: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_level">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>DEBUG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>INFO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WARNING</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ERROR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CRITICAL</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>211</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_instruments">
    <item>
     <widget class="QCheckBox" name="checkBox_PRESSURE">
      <property name="text">
       <string>Pressure controller</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_PUMP">
      <property name="text">
       <string>Syringe pump</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_FLOW_A">
      <property name="text">
       <string>Flow A</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_FLOW_B">
      <property name="text">
       <string>Flow B</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_VALVE2x3">
      <property name="text">
       <string>Double 3-way valve</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_E_VALVE1">
      <property name="text">
       <string>Electrovalve 1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_E_VALVE2">
      <property name="text">
       <string>Electrovalve 2</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_P_CODE">
      <property name="text">
       <string>Picarro Code</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_STATE">
      <property name="text">
       <string>State</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_HUMDEP">
      <property name="text">
       <string>Humdep</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_HUMGEN">
      <property name="text">
       <string>HumGen</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_T_SWITCH">
      <property name="text">
       <string>Temperature Switch</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_T_SENSOR">
      <property name="text">
       <string>Temperature Sensor</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_P_SWITCH">
      <property name="text">
       <string>Pressure Pump Switch</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
